terraform/deployments/govuk-publishing-platform/app_router.tf

```diff
@@ -59,14 +59,14 @@ module "router" {
       # BACKEND_URL_canary-frontend         = module.canary-frontend.virtual_service_names[0]
       # BACKEND_URL_collections             = module.collections.virtual_service_names[0]
       # BACKEND_URL_contacts-frontend       = module.contacts-frontend.virtual_service_names[0]
-      BACKEND_URL_content-store = module.content_store.virtual_service_names[0]
+      BACKEND_URL_content-store = "http://${module.content_store.virtual_service_names[0]}"
       # BACKEND_URL_designprinciples        = module.designprinciples.virtual_service_names[0]
       # BACKEND_URL_email-alert-frontend    = module.email-alert-frontend.virtual_service_names[0]
       # BACKEND_URL_email-campaign-frontend = module.email-campaign-frontend.virtual_service_names[0]
       # BACKEND_URL_external-link-tracker   = module.external-link-tracker.virtual_service_names[0]
       # BACKEND_URL_feedback                = module.feedback.virtual_service_names[0]
       # BACKEND_URL_finder-frontend         = module.finder-frontend.virtual_service_names[0]
-      BACKEND_URL_frontend = module.frontend.virtual_service_names[0]
+      BACKEND_URL_frontend = "http://${module.frontend.virtual_service_names[0]}"
```

Comment:
I guess with https://github.com/alphagov/govuk-infrastructure/pull/226, the array needs to be removed.